packages/protocol/scripts/bash/init_network.sh

```diff
@@ -6,4 +6,4 @@ set -euo pipefail
 # Flags:
 # -n: Name of the network to migrate to
 
-yarn run migrate -r "$@"
+yarn run migrate -r "$@" && yarn run set_block_gas_limit "$@"
```

Comment:
Would it make sense to have this be a last migration script instead of a new independent script?

Reply:
Rationale behind a separate script was that truffle will send final transactions to the Migrations.sol contract after every migration, and the gas amount truffle uses for all contracts is not easily configured. So for migrations it's configured to allow for 20M gas per tx, and after the gas limit is lowered, any further truffle txs will fail.